src/services/pcn-iptables/datamodel/iptables.yang

```diff
@@ -11,6 +11,12 @@ module iptables {
 
   uses "polycube-standard-base:standard-base-yang-module";
 
+  polycube-base:service-description "Iptables-compatible Service";
+  polycube-base:service-version "2.0";
+  polycube-base:service-name "iptables";
+  polycube-base:service-min-kernel-version "4.14.0";
```

Comment:
Isn't the min-kernel-version be "4.15.0"?

https://github.com/polycube-network/polycube/blob/master/Documentation/installation.rst#updating-linux-kernel-to-v415

Reply:
The min kernel version in the YANG refers to the one that is actually required to run this service. The link you point out is the minimum version required to run the entire framework, which is actually max(version_required_by_each_individual_service).
So, this should stay 4.14 if I'm not mistaken.
@mbertrone : do you confirm?